test/framework/src/main/java/org/elasticsearch/test/MockLogAppender.java

```diff
@@ -31,12 +33,35 @@
 /**
  * Test appender that can be used to verify that certain events were logged correctly
  */
-public class MockLogAppender extends AbstractAppender {
+public class MockLogAppender {
 
+    private static final Logger logger = LogManager.getLogger(MockLogAppender.class);
```

Comment:
I don't understand why do we need this. What I'm missing?

Reply:
Leftover from debugging, will remove